### ISSUE TITLE
DeathReason always 255

### ISSUE THREAD

**Issue by sentik** [ikkentim/SampSharp#217]
I have a problem with the death of the player, DeathReason is always equal to 255. I checked all builds of the 7th version, the bug also manifests. I also checked on different versions of the GDK and the versions builded manually.

Of the plugins I use only Streamer.

**Comment by ikkentim** [comment]
I've checked - reason 255 was missing from the definitions list. 255 means suicide. How did you cause these deaths? Are you sure it should cause a different death reason? In the sa-mp server log it says `[death] ikkentim died 255` somewhere. This is logged by sa-mp, not sampsharp. So if it says 255 here it is actually suicide and not a sampsharp bug.

I've added the `Weapon.Suicide` value in 1981e08bba3af847f3f95afc1381d2dde55543eb. This change will be available in the next 0.8 alpha build.
